CMakeLists.txt

```diff
@@ -9,6 +9,9 @@ set(DEPS_PREFIX "${CMAKE_CURRENT_SOURCE_DIR}/.deps/usr" CACHE PATH "Path prefix
 list(INSERT CMAKE_PREFIX_PATH 0 ${DEPS_PREFIX})
 set(ENV{PKG_CONFIG_PATH} "$ENV{PKG_CONFIG_PATH}:${DEPS_PREFIX}/lib/pkgconfig")
 
+# used for check_c_compiler_flag
+include(CheckCCompilerFlag)
```

Comment:
for reference, this was added to line 120 in https://github.com/Pyrohh/neovim/commit/2b4cbbebf478dd949b45f23996c7dd713ee562f3#diff-af3b638bc2a3e6c650974192a53c7291R120 

so the line 120 instance should be removed in this PR.